### DISCUSSION TITLE
美國速食店導入炸物機器人，炸薯條、洋蔥圈效率大勝人類

### DISCUSSION THREAD

**Discussion by Yunhans** [1101-GS4719-C/Python-Community-GS4719-C#763] [科技新知]
[新聞連結](https://tw.news.yahoo.com/美國速食店導入炸物機器人-炸薯條-洋蔥圈效率大勝人類-020057784.html)
[https://www.reuters.com/technology/want-fries-with-that-robot-makes-french-fries-faster-better-than-humans-do-2022-10-04/](https://www.reuters.com/technology/want-fries-with-that-robot-makes-french-fries-faster-better-than-humans-do-2022-10-04/)
看到這個新聞讓我覺得以後速食店是不是連櫃檯幫我們結帳的服務生都要變成機器人了，現在點餐也是用機器人，以後沒辦法去麥當勞打工了。

**Comment by danwan1027** [comment]
AI 正在改變我們的工作型態，太單調的工作終究會被取代

**Comment by a45320614** [comment]
現在已經有新型態的無人商店出現了，我認為下一步很有可能就是無人餐廳呢，畢竟餐飲業的工作都是重複性高且容易被替代的，訓練新人常常需要很多時間

**Comment by Chiway1229** [comment]
隨著AI科技日益月新，越來越多低專業性的工作會開始被機器人取代了
可能同時也會影響整個社會結構呢
不過有更先進的社會對我這個學生固然是好事

**Comment by williamyen0623** [comment]
中午沒吃，這篇文讓我突然想吃麥當勞了，讓我有動力走路消耗卡路里，再補充營養，優質文章推推！

**Reply by Yunhans** [reply]
我比較喜歡摩斯漢堡

**Reply by evanchou1204** [reply]
我是麥當勞派，但漲價了，哭哭

**Comment by manipulateed** [comment]
正是如此，才要選擇被取代率低的行業😢。

**Comment by Pietra1226** [comment]
人類厲害的地方是可以有創新的思考模式，或許在AI尚未普及前，將單一性質高的產業轉型，而無法轉型的再留給機器人

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
不知道如果未來人類都被機器取代的話 會發生什麼問題

**Comment by evanchou1204** [comment]
如果把單調的工作給機器人做，人就可以發揮特長，多多去做比較有創造力、機器人做不到的事。

**Comment by Zoe06060606** [comment]
隨者時代的變遷 科技的發展日新越意 工作被取代也早已不是新聞了 該是時後培養好自己的專業了

**Comment by Zoe06060606** [comment]
總覺得這樣其實挺好的，將SOP完全精準執行就不會有分店好吃不好吃的差異了

**Comment by Jasper525** [comment]
這樣以後速食店徵人可能就變成徵設備工程師了~

**Comment by bonny123865** [comment]
感覺這種機器人目前只有很大的餐飲連鎖才會花成本去使用
但慢慢地未來都會替換掉一些餐飲人員

**Comment by Kotori0501** [comment]
感謝分享
現在的AI技術越來越進步，有很多職業都被漸漸的取代而去，因此能多學一項專業就多學一點吧

**Comment by evanchou1204** [comment]
確實炸薯條、點餐這種單調的工作交給機器人做會快很多，也更節省人力成本。

**Comment by ching020721** [comment]
這種取代性高的產業本來就會找能更省成本的方式進行，因此要做的是精進自己，成為不被AI取代的那個

**Comment by wangtyphoon** [comment]
或許有一天ai能取代的不只是重複性的工作，連創意也無人能及，那樣的話應該算是科技奇點了

**Comment by ruthchen1020** [comment]
哪天發電廠被炸了，我們就沒有薯條吃了@@
所以我們還是要學會煮自己東西。

**Comment by yeh-alex** [comment]
最近已經有自動收盤子的機器人了，雖然目前不可能完全自動，但未來可能真的店裡只有老闆一個真人了

**Comment by peggy0329** [comment]
印象中，日本已經有24小時自助拉麵店了，全透過機器人操作，人們只要進去店裡拿取拉麵就可以坐在裡面吃了
我覺得這樣的型態蠻適合凌晨需要吃宵夜的人
但如果是日常晚餐，感覺這樣就沒什麼溫馨的感覺

**Comment by Evan0979** [comment]
至少這樣子每次點的薯條都會一樣好吃??

**Comment by Zoe06060606** [comment]
Ai遲早會取代人類 由其是單一性值的工作一定會被取代

**Comment by CPCpyt** [comment]
重複單一性的工作被取代是早晚的事，
因此我們要做的是精實自己，不要讓自己容易被取代！

**Comment by zephyrxin01** [comment]
我覺得還是要看人力成本高於機器人成本，這個作法才會被普及，至少現階段感覺維護一台機器人的成本可能還是相對來說比較高，如果有店家推行應該也只是為了噱頭。

**Comment by alstinlin** [comment]
撇除塵本的問題，以機器人取代人力感覺蠻合適的，一方面可以精準地控制炸東西的時間，一方面可以減少出錯而導致的損失，但我認為服務員的腳色應該還是需要由人來負責，才能滿足客人不同的需求

**Comment by sumairu0727** [comment]
第一次工業革命也造成很多工人失業，但長久來看也沒有造成社會對工作人員需求量下降，是因為人類本來就應該往更具有價值的工作前進而不是就在原地做著那些低附加價值的工作，所以我也希望這些低技術高重複性的工作可以盡快被機器人取代。畢竟台灣目前缺的人力是高階人力，也就是需要高技術性的工作，所以盡快讓這些低技術工作被機器人取代我覺得對社會是有好處的。

**Comment by Brownnnn78** [comment]
如果每一次吃麥當勞的薯條都是同一種脆度和鹽量固定的話真的很希望能夠交給機器人製作，這類大型的連鎖店能夠減少因人為因素造成的食物難吃我覺得應該替代掉，讓員工變成輔助的角度。不然有時候吃到不好吃的炸雞很難過。

**Comment by Elaine2899** [comment]
我認為這正是人類思考新型職業型態的時候，當許多低專業的工作將被機器人取代，那麼什麼樣的職業才會不被取代呢?我想是與情感相關的職業，現代人壓力漸大、越來越少有時間與能力發洩壓力，這樣的情況下是不是我們可以開創新的產業、職業與情緒相關呢?我想或許這是一個有可能的未來。

**Comment by PaulSu0905** [comment]
不久前還在開玩笑說別人被二一之後要去麥當勞炸薯條，看來那位同學的未來工作正在不斷減少中。

**Comment by poweichen00** [comment]
在未來也會有越來越多的工作被取代，希望我之後不要被機器人取代掉。

**Comment by Evening218** [comment]
我覺得這樣的確可以提高餐飲業製作餐點以及服務的效率,而且也可以更精確的掌控出餐品質,但我還是不希望未來所有餐廳都變成全自動化,而且萬一機器出問題就整間店停擺了

**Comment by youtinglin** [comment]
可以讓社會運作更有效率，將人力放在更需要的地方，但是全自動化也會有它的缺點，比如說會缺乏人性化的一些感受。

**Comment by ashleyy-huang** [comment]
現在去麥當勞都是由點餐機點餐，雖然比較方便且快速，但感覺少了人與人的親切感，感覺甚麼都很制式化，未來一定還會有很多被取代的工作，希望人類可以做的事不會都被機器人取代。

**Comment by zhiron544** [comment]
我認為這是一個不可避免的趨勢，但也有利有弊。一方面，機器人可以提高效率和品質，減少人力成本和錯誤，也可以讓人類有更多時間和空間去發揮創意和專長。
